src/main/java/org/terasology/Lakes/LakeProvider.java

```diff
@@ -68,35 +82,35 @@ public void process(GeneratingRegion region) {
         region.setRegionFacet(LakeFacet.class, lakes);
     }
 
-    protected boolean checkGradient(Vector3i pos, BaseFieldFacet2D facet){
+    protected boolean checkGradient(Vector3i pos, BaseFieldFacet2D facet) {
```

Comment:
I'm wondering whether we could use something from https://github.com/Terasology/SurfaceFacets for this check? 🤔 might be an improvement for the future though...

Reply:
well yeah the way gradient is checked is specific to a facet, so that's an issue...